examples/biosignal-erp/plot_embedding_MEG.py

```diff
@@ -85,10 +89,23 @@
 plt.show()
 
 
+###############################################################################
+# TNSE
+# ------------------------------
+
+plot_embedding(covs, ytest, metric="riemann", embd_type="TSNE",
```

Comment:
For tSNE, we could select only 2 classes on the 4 from this dataset, and use 2 components from XDAWN. In this case, we should generate a new set of cov mats. This should largely decrease the computational time for doc building.